sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/CheckFileLicenseTest.cs

```diff
@@ -78,7 +78,8 @@ public class CheckFileLicenseTest
  \* along with this program; if not, write to the Free Software Foundation,
  \* Inc\., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA\.
  \*/";
-        private const string MultiSingleLineRegexHeader = "//-{5}\r\n// MyHeader\r\n//-{5}";
+        private const string MultiSingleLineRegexHeader1 = "//-{5}\r\n// MyHeader\r\n//-{5}";
+        private const string MultiSingleLineRegexHeader2 = "//-{5}.+// MyHeader.+//-{5}";
```

Comment:
and this could be `MultiLineRegexWithDot`

Reply:
I find "MultiSingleLineRegexHeader" quite confusing so it's a good time to improve names :)